frame/ethereum/src/lib.rs

```diff
@@ -74,6 +74,11 @@ decl_storage! {
 		TransactionStatuses: map hasher(blake2_128_concat) H256 => Option<TransactionStatus>;
 		Transactions: map hasher(blake2_128_concat) H256 => Option<(H256, u32)>;
 	}
+	add_extra_genesis {
+		build(|_config: &GenesisConfig| {
+			<Module<T>>::store_block(T::BlockNumber::from(0));
```

Comment:
Only issue with this is that if the module is added after genesis, then it won't work, but I guess they would need migration scripts anyway.